lib/shared/config/customize.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var config = require('./');
+var merge = require('lodash.merge');
+var isString = require('lodash.isstring');
+
+// If project dir and cwd are same, the gulprc file in the directory is loaded
+// twice continuously. This function is to avoid it.
+var lastLoadedFile = null;
+function isNotLastLoaded(file) {
+  if (lastLoadedFile === file) {
+    return false;
+  }
+  lastLoadedFile = file;
+  return true;
+}
+
+module.exports = function(configFiles) {
+  configFiles.filter(isString).filter(isNotLastLoaded).forEach(function(file) {
+    merge(config, require(file));
```

Comment:
I think `config` has to be singleton, because it has to be accessible for every modules in `gulp-cli`.

Or should `config` be passed to `lib/versioned/*/index.js` as an argument?